configure.in

```diff
@@ -1145,6 +1142,9 @@ PKG_CHECK_MODULES([FREETYPE2],  [freetype2],
 PKG_CHECK_MODULES([TAGLIB],  [taglib >= 1.8],
   [INCLUDES="$INCLUDES $TAGLIB_CFLAGS"; LIBS="$LIBS $TAGLIB_LIBS"],
   AC_MSG_ERROR($missing_library))
+PKG_CHECK_MODULES([ICU_UC], [icu-uc >= 54.1],
```

Comment:
Technically we can use v48. The newer stuff is only needed on Darwin platforms.

There are some issues with this on Ubuntu though, the packaged version they ship does not include pkg-config files and it's incorrectly built so we have to check for the specific version we want. I'm not really a Linux guy so there might be a better way to go about this but this is the Changes I made to configure.in
https://gist.github.com/Paxxi/2be7d0cd38c1505a9d76#file-gistfile1-mak